hist/hist/src/TH1.cxx

```diff
@@ -2505,12 +2505,15 @@ void TH1::ClearUnderflowAndOverflow()
 ///  The resulting integral is normalized to 1.
 ///  If the routine is called with the onlyPositive flag set an error will
 ///  be produced in case of negative bin content and a NaN value returned
+///  \param option (optional) Set it to "width" if your non-uniform bin contents
+///  represent a density rather than counts. In that case, we internally multiply
+///  by the bin width/area so that fIntegral is in the "counts" and not on the original "density" domain.
```

Comment:
We should maybe first explain what the default does, and then go to the options.
This is what I understand the function to do (please correct if wrong):
```suggestion
///  \param onlyPositive If set to true, an error will be produced and NaN will be returned
///  when a bin with negative number of entries is encountered.
///  \param option
///  - `""` (default) Compute the cumulative density function.
///  - `"width"` When using a histogram that has been scaled to represent densities instead of event counts, 
///     this options multiplies the densities with the bin width.
///     The cumulative function in that case represents counts instead of probabilities.
```